corehq/apps/userreports/transforms/custom/date.py

```diff
@@ -4,7 +4,10 @@
 
 
 def get_month_display(month_index):
-    return calendar.month_name[int(month_index)]
+    try:
+        return calendar.month_name[int(month_index)]
+    except Exception as e:
```

Comment:
nit: this is a broad exception case.  Could be more explicit by catching ValueError if month_index== "", KeyError if month_index == 13, etc.